core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -641,9 +641,14 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
         siteAnalyticsService.registerSavePlayableExplorationEvent(
           explorationData.explorationId);
       }
-
       $scope.isSaveInProgress = true;
-      explorationData.save(changeList, commitMessage, function() {
+      explorationData.save(changeList, commitMessage, function(response) {
```

Comment:
Shouldn't this function be part of the errorCallback? It seems strange that you would log "Changes to this exploration were saved successfully" and then show a failure modal.